manual/manual/refman/exten.etex

```diff
@@ -620,92 +618,96 @@ Each implementation is a structure that we can encapsulate as a
 first-class module, then store in a data structure such as a hash
 table:
 \begin{caml_example*}{verbatim}
-        module type DEVICE = sig [@@@ellipsis] end
-        let devices : (string, (module DEVICE)) Hashtbl.t = Hashtbl.create 17
+type picture = unit[@ellipsis]
+module type DEVICE = sig val draw : picture -> unit [@@ellipsis] end
```

Comment:
I would propose to have a separate ellipsis `[@@@ellipsis]` to have
```OCaml
module type DEVICe = sig
val draw:picture -> unit
...
end
```
rather than just `sig ... end` since it gives a clearer purpose to `picture`.